Storage/SuggestedSites.swift

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCGLogger
+import UIKit
+import Shared
+
+private let log = XCGLogger.defaultInstance()
+
+public class SuggestedSite: Site {
+    public let wordmark: Favicon
+    public let backgroundColor: UIColor
+
+    let trackingId: Int
+    init(json: JSON) {
+        let colorString = json["bgcolor"].asString!
```

Comment:
Nit: Are we converting a hex value to UIColor here? Could we put this into a UIColor extension under Utils for reuse else where?